lib/conversation/replies.js

```diff
@@ -95,7 +104,7 @@ function renderContinueConversationMsg(fn) {
         // Store current campaign to continue conversation in subsequent messages.
         // TODO: Should this be handled more eloquently?
         args.req.user.current_campaign = args.req.campaignId;
```

Comment:
Hmm, what about if we create an schema method to save the current campaign, just like how saving the last outbound template works. Then we could use a Promise.all to proceed with the reply? Not a blocker for merging.